railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_7_2.rb.tt

```diff
@@ -28,8 +28,8 @@
 # If `enqueue_after_transaction_commit` is set to `:default`, the queue adapter
 # will define the behaviour.
 #
-# Note: Active Job backends can disable this feature. This is generally used by
-# backends that use the same database than Active Record as a queue, hence they
+# Note: Active Job backends can disable this feature. This is generally done by
+# backends that use the same database as Active Record as a queue, hence they
```

Comment:
This also makes this note consistent with the one in `guides/source/configuring.md` that was updated in #51489.